administration/src/application/components/ApplyController.tsx

```diff
@@ -103,6 +121,7 @@ const ApplyController = () => {
             onSubmit={submit}
             loading={loading || loadingPolicy}
             privacyPolicy={policyData?.dataPolicy.dataPrivacyPolicy ?? ''}
+            regionData={regions}
```

Comment:
Most likely, you should pass the regions inside the options prop of ApplicationForm and PersonalDataForm (and not inside their Additional Props)